src/watchkit.cs

```diff
@@ -857,6 +857,10 @@ interface WKExtensionDelegate {
 		[Export ("handleUserActivity:")]
 		void HandleUserActivity ([NullAllowed] NSDictionary userInfo);
 
+		[Watch (3,2)]
+		[Export ("handleActivity:")]
+		void HandleActivity (NSUserActivity userActivity);
```

Comment:
what's the doc saying about this API ? it seems Apple just could not reuse `handleUserActivity:` and went with `handleActivity:` accepting an `NSUserActivity`.

It might make more sense (if the description are identical/close) to make this an overload of `HandleUserActivity`.